content/en/user-guide/aws/ec2/index.md

```diff
@@ -501,22 +501,23 @@ $ virsh pool-refresh default
 Pool default refreshed
 </disable-copy>
 $ virsh vol-list --pool default
+<disable-copy>
  Name                                    Path
 --------------------------------------------------------------------------------------------------------
-
  ami-1234abcd                            /var/lib/libvirt/images/ami-1234abcd
+</disable-copy>
 {{< /command >}}
 
 Only the images that follow the above naming scheme will be recognised by LocalStack as AMIs suitable for launching virtualised instances.
 These AMIs will also have the resource tag `ec2_vm_manager:libvirt`.
 
 {{< command >}}
-awslocal ec2 describe-images --filters Name=tag:ec2_vm_manager,Values=libvirt
+$ awslocal ec2 describe-images --filters Name=tag:ec2_vm_manager,Values=libvirt
 {{< /command >}}
 
 ### Instances
 
-Virtualised instances can be launched with `RunInstances` operation and specifying a compatible AMI.
+Virtualized instances can be launched with `RunInstances` operation and specifying a compatible AMI.
```

Comment:
nit: We should be consistent with either American English or British English. Eg. in line `511` we are using `virtualised`.